config/config.go

```diff
@@ -415,6 +415,7 @@ func get(ctx *cli.Context) (*Config, error) {
 			AccessKeyID:              ctx.String("s3.access_key_id"),
 			SecretAccessKey:          ctx.String("s3.secret_access_key"),
 			DisableSSL:               ctx.Bool("s3.disable_ssl"),
+			UpdateTimestamps:         ctx.Bool("s3.update_timestamps"),
```

Comment:
If we end up keeping this option (instead of making it always-on), we'll need to update the README.md file with the new help text and sample config file.

Reply:
Let me know what you think about leaving it opt-in, then I will update documentation if is stays this way 👍